deploy/04-operatorgroup.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operators.coreos.com/v1
```

Comment:
this is generic enough I'm not sure about it entirely, but if it is only needed for the postgresql subscription below, it too needs to be moved to a test specific deploy dir